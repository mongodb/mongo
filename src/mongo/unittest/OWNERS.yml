version: 1.0.0
filters:
  - "*":
    approvers:
      - 10gen/server-programmability
  # Although golden testing is a generic unit testing utility that can be used by any team, it was
  # originally built by the query team and as of this writing is primarily used for query-related
  # unit tests.
  - "golden_test*":
    approvers:
      - 10gen/query-optimization
  - "ensure_fcv.h":
    approvers:
      - 10gen/server-fcv
