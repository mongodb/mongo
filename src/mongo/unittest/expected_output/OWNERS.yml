version: 1.0.0
filters:
  # This directory contains expected output for golden data self-tests. Although golden testing is a
  # generic unit testing utility that can be used by any team, it was originally built by the query
  # team and as of this writing is primarily used for query-related unit tests.
  - "*":
    approvers:
      - 10gen/query-optimization
  - "OWNERS.yml":
    approvers:
      - 10gen/query-optimization-staff-leads
